src/Traits/AdapterHelper.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\Shopware6Common\Traits;
+
+use FINDOLOGIC\Export\Data\Attribute;
+use FINDOLOGIC\Shopware6Common\Export\Config\PluginConfig;
+use FINDOLOGIC\Shopware6Common\Export\Utils\Utils;
+use Vin\ShopwareSdk\Data\Entity\PropertyGroupOption\PropertyGroupOptionCollection;
+
+trait AdapterHelper
+{
+    protected PluginConfig $pluginConfig;
+
+    /**
+     * @return Attribute[]
+     */
+    protected function getPropertyGroupOptionAttributes(
+        PropertyGroupOptionCollection $collection
+    ): array {
+        $attributes = [];
+
+        foreach ($collection as $propertyGroupOptionEntity) {
+            if (
+                $propertyGroupOptionEntity->group
+                &&
+                !$propertyGroupOptionEntity->group->filterable
+            ) {
+                continue;
+            }
```

Comment:
Please keep the code as it was before.

```suggestion
            $group = $propertyGroupOptionEntity->group;
            if ($group && !$group->filterable) {
                continue;
            }
```